feeds/ucentral/ucentral-schema/files/usr/share/ucentral/wifi_max_user.uc

```diff
@@ -12,5 +15,13 @@ function phy_get() {
 	return res;
 }
 
+let board_name = board_info.board_name;
+
+switch(board_name) {
+case 'edgecore,eap112':
+        max_assoc = 128;
+        break;
+}
+
```

Comment:
rtrim(fs.readfile('/tmp/sysinfo/board_name'), '\n');   would be a lot easier than using ubus